cytosnake/utils/cyto_paths.py

```diff
@@ -76,16 +76,13 @@ def get_cytosnake_package_path() -> pathlib.Path:
     Raises
     ------
     FileNotFoundError
-        Raised if the `.git` folder is not found.
+        Raised if CytoSnake Package is not found
     """
 
-    # get location of this file
-    # -- check if he ".git" folder exists
+    # get parent path of the `__init__.py` module and check it exists
     project_path = pathlib.Path(cytosnake.__path__[0]).parent
```

Comment:
Seeing this comment and line of code made me wonder about how these get used throughout CytoSnake. This led me to the following line: https://github.com/WayScience/CytoSnake/blob/67ae1aee00d659611c6e236fffb73b854377dd62/cytosnake/utils/cytosnake_setup.py#L72 

When "source" and "target" language is used there is usually an implied directionality of flow (or a sort of "gravity") from source -to-> target. Observing the line above, and understanding CytoSnake as a "source", I get the feeling CytoSnake may be using source as a target which could present challenges for the audience. For example, users may wish to install CytoSnake in a different location than their data can be stored.

There could be an opportunity now or later to distinguish the target as a separate location (optionally) decoupled from CytoSnake module storage. If there are dependencies from CytoSnake which incurred this design choice, consider also making those a part of the target or utilizing a temporary directory to assist with how the data are created in a way that allows for decoupled sources and targets.

Reply:
The code you have shown is part of a function that will transfers files within the `CytoSnake` package into the `ProjectDirectory`. The code presents the `CytoSnake` package as a source and the `ProjectDirectory` as a the target. 

But you are correct. Now looking at the code, it does seem difficult to understand. I know for a fact once we make `CytoSnake` `pip` and `conda` installable, this will definitely  be updated. 

Thank you for the insight.